src/test/java/org/prebid/server/ApplicationTest.java

```diff
@@ -497,8 +1186,8 @@ public void auctionShouldRespondWithBidsFromDifferentExchanges() throws IOExcept
         assertThat(response.header("Access-Control-Allow-Credentials")).isEqualTo("true");
         assertThat(response.header("Access-Control-Allow-Origin")).isEqualTo("http://www.example.com");
 
-        final String expectedAuctionResponse = auctionResponseFrom(jsonFrom("auction/test-auction-response.json"),
-                response, "bidder_status.find { it.bidder == '%s' }.response_time_ms");
+        final String expectedAuctionResponse = auctionResponseFrom(jsonFrom("auction/rubicon&appnexus/test-auction-rubicon-appnexus-response.json"),
+                response, "bidder_status.find { it.bidder == '%s' }.response_time_ms", exchanges);
```

Comment:
Is it possible to get rid of repetition of `"bidder_status.find { it.bidder == '%s' }.response_time_ms"` and `"ext.responsetimemillis.%s"` in calls to `auctionResponseFrom`?